image/verify/src/verifier.rs

```diff
@@ -345,6 +415,8 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
             caliptra_cfi_lib::cfi_assert_eq_12_words(&expected, &actual);
         }
 
+        // [TODO] Verify active public key's digest from the descriptor hash list.
```

Comment:
Does this still need to be done in this PR?

Reply:
I'll do that as a follow up change when updating the image for MLDSA keys.